editor/src/templates/editor-entry-point.tsx

```diff
@@ -7,6 +7,8 @@
 // import feature switches so they are loaded before anything else can read them
 import '../utils/feature-switches'
 
+import '../vite-shims'
```

Comment:
is there a reason why this is not the top import?

Reply:
Actually no, it should be

Reply:
Done